articles/active-directory/develop/quickstart-v2-javascript.md

```diff
@@ -72,39 +74,40 @@ Select the option that's suitable to your development environment:
 
 * To run the project with a web server by using Node.js, [download the core project files](https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/archive/quickstart.zip). To open the files, use an editor such as [Visual Studio Code](https://code.visualstudio.com/).
 
-* (Optional) To run the project with the IIS server, [download the Visual Studio project](https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/archive/vsquickstart.zip). Extract the zip file to a local folder (for example, *C:\Azure-Samples*).
-
 #### Step 3: Configure your JavaScript app
 
 > [!div renderon="docs"]
-> In the *JavaScriptSPA* folder, edit *index.html*, and set the `clientID` and `authority` values under `msalConfig`.
+> In the *JavaScriptSPA* folder, edit *authConfig.js*, and set the `clientID` and `authority` values under `msalConfig`.
 
 > [!div class="sxs-lookup" renderon="portal"]
-> In the *JavaScriptSPA* folder, edit *index.html*, and replace `msalConfig` with the following code:
+> In the *JavaScriptSPA* folder, edit *authConfig.js*, and replace `msalConfig` with the following code:
 
 ```javascript
-var msalConfig = {
+
+  // Config object to be passed to Msal on creation
+  const msalConfig = {
     auth: {
-        clientId: "Enter_the_Application_Id_here",
-        authority: "https://login.microsoftonline.com/Enter_the_Tenant_info_here",
-        redirectUri: "http://localhost:30662/"
+      clientId: "Enter_the_Application_Id_Here",
+      authority: "Enter_the_Cloud_Instance_Id_HereEnter_the_Tenant_Info_Here",
```

Comment:
should there be a "/" between the cloud instance id and the tenant info?

Reply:
slash is automatically supplied after each url replacement